pandas/_libs/tslibs/period.pyx

```diff
@@ -1295,6 +1295,25 @@ cdef class _Period(object):
 
     @property
     def second(self):
+        """
+        Get second of the minute component of the Period.
```

Comment:
Say the same thing you are saying in See Also but for second, so literally "Get the second component of the Period"

Reply:
but period I'm bit confusesd because when I ren code it gives that particular second of that minute have

Reply:
Not sure I understand your comment. The below will both return 12, so the minute is irrelevant:

```python
pd.Period("2018-03-11 13:03:12.050000").second
pd.Period("2018-03-11 13:04:12.050000").second
```

Reply:
yup I made changes :+1:  thanks for reveiw